public/content/documentation/how-to-test/components/links-&-buttons.md

```diff
@@ -24,98 +24,105 @@ best practices.
+        <h3>✓ Ensure buttons and links can be activated with the <code>enter</code> key and that buttons can also be activated with the <code>space</code> key</h3>
+        <div class="table-wrapper">
+          <table>
               <thead>
               <th scope="col">
-                  Preferred
+                  Pass
               </th>
               <th scope="col">
-                  Visually disabled but actionable
+                  Fail
               </th>
               </thead>
               <tbody>
               <tr>
               <td>
-                  <button class="Magentaa11y-button Magentaa11y-button--primary" aria-disabled="true">Save</button>
+                  <button class="Magentaa11y-button Magentaa11y-button--primary" data-fn="showAlert">Show alert</button>
+              </td>
+              <td>
+                  <div class="Magentaa11y-button Magentaa11y-button--primary" type="button" tabindex="0" data-fn="showMouseAlert" data-event="onMouseDown">
+                  Show alert
+                  </div>
+              </td>
+              </tr>
+                  <tr>
+              <td>
+                  <a href="/home">Magentaa11y home</a>
               </td>
               <td>
-                  <button class="Magentaa11y-button Magentaa11y-button--primary visually-disabled" tabindex="-1" data-fn="showAlertWhenDisabled">Save</button>
+                  <a tabindex="0" data-fn="goToHome" data-event="onMouseUp" >Magentaa11y home</a>
               </td>
               </tr>
               </tbody>
           </table>
+        </div>
+      </div>
+
+      <div class="how-to-test-checklist-item">
+          <h3>✓ Ensure disabled controls are focusable but not actionable, and have an <code>aria-disabled="true"</code> attribute</h3>
+          <p><strong>Note:</strong> A control that is disabled should not be interactive to any users. Nothing should happen when activated.</p>
+          <div class="table-wrapper">
+            <table >
+                <thead>
+                <th scope="col">
+                    Preferred
+                </th>
+                <th scope="col">
+                    Visually disabled but actionable
+                </th>
+                </thead>
+                <tbody>
+                <tr>
+                <td>
+                    <button class="Magentaa11y-button Magentaa11y-button--primary" aria-disabled="true">Save</button>
+                </td>
+                <td>
+                    <button class="Magentaa11y-button Magentaa11y-button--primary visually-disabled" tabindex="-1" data-fn="showAlertWhenDisabled">Save</button>
+                </td>
+                </tr>
+                </tbody>
+            </table>
+          </div>
       </div>
 
       <div class="how-to-test-checklist-item">
           <h3>✓ Ensure all links and buttons have clear labels and that all graphical controls have accurate <code>aria-label</code> attributes</h3>
```

Comment:
This table is the only one that seems wonky (even at a normal breakpoint) all others do still work at a small break point; happy to retest!